File: Algorithms/Recursion_Algorithm/Recursion_Algorithm.cpp

#include <bits/stdc++.h> 
using namespace std; 

void printFun(int test) 
{ 
	if (test < 1) 
		return; 
	else { 
		cout << test << " "; 
		printFun(test - 1); // statement 2 
		cout << test << " "; 
		return; 
	} 
} 

int main() 
{ 
	int test = 3; 
	printFun(test); 
} 

File: Algorithms/Recursion/Recursion_Algorithm.py

def printFun(test): 

	if (test < 1): 
		return
	else: 
	
		print( test, end = " ") 
		printFun(test-1) # statement 2 
		print( test, end = " ") 
		return
	

test = 3
printFun(test) 

